### PR TITLE
Update guestbook-logs-metrics-with-elk.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with english version

### CODE REVIEW

File: content/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ This tutorial builds upon the [PHP Guestbook with Redis](/docs/tutorials/statele
 将和 Guestbook 一同部署在 Kubernetes 集群中。
 Beats 收集、分析、索引数据到 Elasticsearch，使你可以用 Kibana 查看并分析得到的运营信息。
 本示例由以下内容组成：
-
+* 一个运行的
+[使用 Redis 部署 PHP Guestbook 教程](/zh/docs/tutorials/stateless-application/guestbook)
```
Comment (tengqm) [inline content/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md:39]:
```suggestion
* [带 Redis 的 PHP Guestbook 教程](/zh/docs/tutorials/stateless-application/guestbook)
  的一个实例部署
```

注意缩进。

Comment (guzj11) [reply content/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md:39]: "PHP Guestbook with Redis" is translated to "使用 Redis 部署 PHP Guestbook" in the other part of the previous translation

Comment (tengqm) [reply content/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md:39]: 理解。那就是说，现在的翻译还有其它问题需要修复。

**Pull request merged**